admin-manual/maintenance/cli-tools.rst

```diff
@@ -1114,6 +1114,45 @@ AtoM, see: :ref:`manage-static-pages`.
    For developers interested in seeing the code where slugs are handled in
    AtoM, see ``/lib/model/QubitSlug.php``
 
+.. _cli-rename-slugs:
+
+Rename slugs
+============
+
+This command line tool can be used to rename existing slugs in AtoM. It offers
+the ability to either rename a single slug, or to run a batch rename using a
+csv file.
+
+.. code:: bash
+
+   php symfony tools:rename [<old-slug> <new-slug>] [--csv=<path-to-file.csv>]
+
+**Task options**
+
+.. image:: images/cli-rename-slug.*
+   :align: center
+   :width: 70%
+   :alt: An image of the CLI options when invoking the tools:rename-slug command
+
+By entering ``php symfony help tools:rename-slug`` into the command-line, you
+see the options and descriptions available on this tool, as pictured above.
+
+The ``--application``, ``--env``, and ``connection`` options **should not be
+used** - AtoM requires the uses of the pre-set defaults for symfony to be
+able to execute the task.
+
+When used without any other options, the rename slugs task expects two
+arguments: an existing slug, and a new name for the existing slug.
+
+The ``--csv`` option can be used to provide a CSV file to use for a batch
+import. The supplied CSV file only needs two columns: an oldSlug column
+(containing the existing slugs), and a newSlug column. Here is an example:
+
+.. image:: images/cli-rename-example.*
+   :align: center
+   :width: 70%
+   :alt: An image of an example CSV file used for a rename-slug command
+
```

Comment:
Not totally necessary but: I have found that, with a user base that is mostly domain users (i.e. archivists w/o much CLI experience), it's good to give concrete examples - otherwise they end up literally using the angle and square brackets you've used in the example syntax above. Here, below your image, might be a good place to give a concrete example now that you've shown a sample CSV. Something like: 

Below is an example of how to run the task using a CSV file: 

```
php symfony tools:rename-slug --csv="path/to/my/new-slugs.csv"
```

Reply:
I also try to link up any related content throughout the docs, so I might also suggest at the end of this section, adding: 

```
.. SEEALSO::
   
   * :ref:`rename-title-slug`
   * :ref:`slugs-in-atom`
   * :ref:`description-permalinks`
   * :ref:`cli-generate-slugs`
 ```

Reply:
Thanks @fiver-watson ! I've added a couple of examples and added links to related docs. Also changed the section title to **Rename slugs via command line** for clarity